source/client/index.js

```diff
@@ -26,10 +27,13 @@ const store = createStoreWithMiddleware(reducer);
 // reduxRouterMiddleware.listenForReplays(store)
 
 ReactDOM.render(
-  <Provider store={store}>
+  <Provider store={ store }>
     <Router history={ browserHistory }>
-      <Route path="/" component={ createApp(React) } />
-      <Route path="/test-data" component={ createTestData(React) } />
+      <Route path='/' component={ createContainer(React) }>
+        <IndexRoute component={ createHome(React) } />
+        <Redirect from='/home' to='/' />
```

Comment:
> I'd like to be able to use them later on with the easy-API as well so I put them in here in the first place. If you think thats not desireable or necessary I can leave it out for now.

It's not necessary here. It should work just fine with the easy API, since it just takes regular react-router routes. All the features of the react-router API should work fine.